components/event-publisher/event-output-adapters/org.wso2.carbon.event.output.adapter.http/src/main/java/org/wso2/carbon/event/output/adapter/http/HTTPEventAdapterFactory.java

```diff
@@ -117,4 +120,15 @@ public OutputEventAdapter createEventAdapter(OutputEventAdapterConfiguration eve
                                                  Map<String, String> globalProperties) {
         return new HTTPEventAdapter(eventAdapterConfiguration, globalProperties);
     }
+
+    private boolean isEnableFormUrlEncoded() {
+
+        Map<String, String> globalProperties = EventAdapterUtil.getGlobalProperties(HTTPEventAdapterConstants.ADAPTER_TYPE_HTTP);
+        boolean enableFormUrlEncoded = false;
+        if (globalProperties.get(HTTPEventAdapterConstants.ENABLE_FORM_URL_ENCODED) != null) {
+            enableFormUrlEncoded = Boolean
+                    .parseBoolean(globalProperties.get(HTTPEventAdapterConstants.ENABLE_FORM_URL_ENCODED));
+        }
+        return enableFormUrlEncoded;
```

Comment:
I think `parseBoolean` handles null as false, you can just do
```suggestion
        return Boolean
                    .parseBoolean(globalProperties.get(HTTPEventAdapterConstants.ENABLE_FORM_URL_ENCODED));
```

Reply:
Fixed